flow/alerting/email_alert_sender.go

```diff
@@ -0,0 +1,100 @@
+package alerting
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/service/ses"
+	"github.com/aws/aws-sdk-go-v2/service/ses/types"
+
+	"github.com/PeerDB-io/peer-flow/peerdbenv"
+)
+
+type EmailAlertSender interface {
+	AlertSender
+}
+
+type emailAlertSenderImpl struct {
```

Comment:
```suggestion
type EmailAlertSender struct {
```
See comment in slack alerter